source/index.css

```diff
@@ -16,8 +16,39 @@ html {
     font-family: box;
 }
 
-canvas {
+canvas, .player-score-list {
     @include aspect-ratio(160, 90);
 }
 
+.player-score {
+    text-align: center;
+    font-weight: bold;
```

Comment:
Let's not make it `font-weight: bold`. I understand that made it look weird with the `text-shadow`, but the trick here is to have multiple shadows.

```css
text-shadow: 0.25vmin 0.25vmin #C44040,
             0.5vmin 0.5vmin #C44040;
```